frontend/src/index.js

```diff
@@ -8,14 +13,19 @@ import "./styles/App.css";
 import { store, persistor } from "./app/store";
 import { PersistGate } from "redux-persist/integration/react";
 
+const cache = new InMemoryCache({});
```

Comment:
are we using both the Apollo cache and redux?

Reply:
My vote is to use Apollo cache for everything in the first version and then reintroduce redux later on if there is a need

Reply:
As long as we still leverage selectors since they store all the data-processing logic, it doesn't really matter what in-memory solution we use. I am more than happy to see all the redux boilerplate exorcised from the app.

Reply:
hmm, right now selectors are used to join data across the different object models (ie: grab Patient data to populate an entry in the Queue). This was designed to interact with a RESTful api that wouldn't necessarily do the joins for us.

Still would be nice to decouple component logic from data processing logic from the raw data.